core/templates/dev/head/admin/Admin.js

```diff
@@ -56,10 +56,10 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
     $http.post($scope.adminHandlerUrl, {
       action: 'revert_config_property',
       config_property_id: configPropertyId
-    }).success(function() {
+    }).then(function() {
       $scope.message = 'Config property reverted successfully.';
       $scope.reloadConfigProperties();
-    }).error(function(errorResponse) {
+    }, function(errorResponse) {
       $scope.message = 'Server error: ' + errorResponse.error;
```

Comment:
Does errorResponse still have an 'error' property? I can't find it [here](https://docs.angularjs.org/api/ng/service/$http).

Also, just to check, have you manually tested the success + error cases at least once? You don't have to do so for every single $http call, but you should definitely do so at least one for each distinct case that arises.